x-pack/test/functional/apps/dashboard/reporting/screenshots.ts

```diff
@@ -29,9 +29,12 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const kibanaServer = getService('kibanaServer');
   const ecommerceSOPath = 'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json';
 
-  // https://github.com/elastic/kibana/issues/102911
-  describe.skip('Dashboard Reporting Screenshots', () => {
+  describe('Dashboard Reporting Screenshots', () => {
     before('initialize tests', async () => {
+      await kibanaServer.uiSettings.replace({
+        defaultIndex: '5193f870-d861-11e9-a311-0fa548c5f953',
+      });
+      await kibanaServer.uiSettings.update({ 'doc_table:legacy': false });
```

Comment:
This line should have stayed down on 218. Looks like it has been moved up, but we only want to update the `doc_table` setting for a single test: "downloads a PDF file with saved search given EuiDataGrid enabled"

Reply:
So I had failures again when I moved the doc table update into single test. I think its got to do with my local. I have updated it and pushed.

Reply:
Thanks @tsullivan